### PR TITLE
Added script to fix ignored keywords

### PR DESCRIPTION
**Pull request by hakonhagland**
See #52 for background.

### CODE REVIEW

File: scripts/src/fodt/fix_ignored.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+                if match := re.match(r"""(\s*<text:p\s+.*?>)(.*?)(</text:p>.*$)""", line, re.DOTALL):
+                    start, txt, end = match.groups()
+                    # Remove span tags from txt
+                    txt = re.sub(r"""<text:span .*?>(.*?)</text:span>""", r"\1", txt)
+                    txt = re.sub(r"""<text:span>(.*?)</text:span>""", r"\1", txt)
+                    if "is ignored by OPM Flow and has no effect on the simulation" in txt:
+                        logging.info(f"Found ignored match in {file}:{line_no}.")
+                        txt = self.replace_ignored(txt, critical)
+                        line = start + txt + end
+                        changed = True
```
Comment (blattms) [inline scripts/src/fodt/fix_ignored.py:46]: Just one question. There is ugly case with `... simulatio</text:span><text:span text:style-name="T2789">n.</text:span>`. That is probably not covered and needs to be done manually. I guess that is ok, though as we cannot anticipate these kind of formatting things.

**Review by blattms — COMMENTED** [review]
This looks really cool. Thanks  a lot.

@lisajulia can you have  a look at the code and merge, please?

File: scripts/src/fodt/fix_ignored.py (hunk 1 above)
Comment (hakonhagland) [reply scripts/src/fodt/fix_ignored.py:46]: > That is probably not covered and needs to be done manually.

@blattms If you mean the case that there is no space between closing and starting tag: `</text:span><text:span>`, it should also work. I tested it now.

File: scripts/src/fodt/fix_ignored.py [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+#    "This keyword is not supported by OPM Flow but would change the results "
+#       if supported so the simulation will be stopped."
+#
+#  - Replacement phrase for non-critical keywords:
+#     "This keyword is not supported by OPM Flow but has no effect on the results so will be ignored."
+#
+@click.command()
+@ClickOptions.maindir(required=False)
+def fix_ignored(maindir: str
+) -> None:
```
Comment (lisajulia) [inline scripts/src/fodt/fix_ignored.py:160]: Is the line break here intentional?

Comment (hakonhagland) [reply scripts/src/fodt/fix_ignored.py:160]: No they are not :) Thanks, for the comment, I will fix this. (I have problems pushing to this branch for some reason, so need to fix this problem first)

Comment (hakonhagland) [reply scripts/src/fodt/fix_ignored.py:160]: > I have problems pushing to this branch for some reason, so need to fix this problem first

I have updated the PR in a new PR, see #149.

**Comment by hakonhagland** [comment]
Replaced by #149.

**Pull request closed without merging**